airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2698,16 +2698,18 @@ def execute(self, context: Any):
             self._handle_job_error(job)
 
             return self.job_id
-        self.defer(
-            timeout=self.execution_timeout,
-            trigger=BigQueryInsertJobTrigger(
-                conn_id=self.gcp_conn_id,
-                job_id=self.job_id,
-                project_id=self.project_id,
-                poll_interval=self.poll_interval,
-            ),
-            method_name="execute_complete",
-        )
+        else:
+            if job.running():
```

Comment:
If the job has completed, can we add some log for the user to tell that query has finished processing? It looks like we silently exit here without any information for the user.

Reply:
+1, in my previous comment, I referred to the fact that it may silently exit with an error, and we won't be aware of it

Reply:
+1